src/yio-interface/integrationinterface.h

```diff
@@ -62,6 +76,15 @@ class IntegrationInterface {
      */
     virtual void sendCommand(const QString& type, const QString& entityId, int command, const QVariant& param) = 0;
 
+    /**
+     * @param type
+     * @param entityId
+     * @param command
+     * @param param
+     */
+    virtual void sendCustomCommand(const QString& type, const QString& entityId, int commandIndex,
```

Comment:
Why is `sendCustomCommand` required what `sendCommand` can't handle?
Couldn't it be integrated in `sendCommand` with additional information in `param`?